package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@curi-bio/mantarray-frontend-components",
-  "version": "0.3.0",
+  "version": "0.4.0",
```

Comment:
this was from my last pull, do I need to add another? I wasn't sure with just testing updates.